src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarReader.cs

```diff
@@ -11,7 +11,7 @@
 namespace System.Formats.Tar
 {
     /// <summary>
-    /// Reads a tar archive from a stream.
+    /// Reads a TAR archive from a stream.
```

Comment:
All of these `tar`s were made `TAR`s.  I very rarely see "tar" capitalized as an acronym, e.g. wikipedia uses "tar" rather than "TAR" when talking about "tar archives":
https://en.wikipedia.org/wiki/Tar_(computing)
We've decided to go a different way?